src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,10 @@
+* {
```

Comment:
Ideally, we should not use `*` selector, as it might have negative impact on performance